backend/src/models/round.model.ts

```diff
@@ -1,12 +1,28 @@
-import { Document, ObjectId, Schema } from "mongoose";
+import { Document, ObjectId, Schema, Types } from "mongoose";
 import { SetupSchema, Setup } from "./setup.model";
+import { Player } from "./player.model";
+
+export enum RoundState {
+  before = "BEFORE",
+  playersChoose = "PLAYERS_CHOOSE",
+  hostChooses = "HOST_CHOOSES",
```

Comment:
It's not the host choosing right? Unless you're just calling The Man™️ host for ease of use

Reply:
Ah right so for all intents & purposes "the host" is "The Man™" in the backend so we don't have to manage extra state

Reply:
Right so the host role is shifted around in the backend? Does this mean that the person who started the original lobby is the first The Man™️?